dashboard/src/containers/HeaderContainer/HeaderContainer.ts

```diff
@@ -23,13 +23,15 @@ function mapStateToProps({
     authenticated,
     namespace,
     pathname,
+    // If autoAuthenticated it's not possible to logout
+    disableLogout: autoAuthenticated,
   };
 }
 
 function mapDispatchToProps(dispatch: ThunkDispatch<IStoreState, null, Action>) {
   return {
     fetchNamespaces: () => dispatch(actions.namespace.fetchNamespaces()),
-    logout: (token: string) => dispatch(actions.auth.logout()),
+    logout: () => dispatch(actions.auth.logout()),
```

Comment:
this change was unrelated to the PR but the linter was complaining about the unused parameter